actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -51,12 +51,12 @@ def path_parameters
       # Invoca Patch
       def strip_string_params!(value_to_strip)
         case value_to_strip
-          when String
-            value_to_strip.strip!
-          when Hash
-            value_to_strip.each{ |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || key.to_sym.not_in?(do_not_strip_string_parameters) }
-          when Array
-            value_to_strip.each{ |value| strip_string_params!(value) }
+        when String
+          value_to_strip.strip!
+        when Hash
+          value_to_strip.each { |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || !do_not_strip_string_parameters.include?(key.to_sym) }
```

Comment:
If `do_not_strip_string_parameters` can be nil, I think this would have a similar problem.
The code used to be:
```
key.to_sym.not_in?(do_not_strip_string_parameters)
```
And this worked fine with nil:
```
[1] pry(main)> :test.not_in?(nil)
true
```
But the updated code uses `include?`:
```
!do_not_strip_string_parameters.include?(key.to_sym)
```
Which complains with nil:
```
[3] pry(main)> !nil.include?(:test)
NoMethodError: undefined method `include?' for nil:NilClass
from (pry):3:in `<main>'
```